01-syntax/05-oop/Vector.js

```diff
@@ -1 +1,25 @@
-// Your code here.
+function Vector(x, y) {
```

Comment:
Use this one 
https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/05-oop/Vector.js#L1